### DISCUSSION TITLE
can imagesharp convert png with alpha channel to tiff?

### DISCUSSION THREAD

**Discussion by moonfanslyj** [SixLabors/ImageSharp#2831] [Q&A] [answered]
hello ：
i try to convert the png which have alpha channel to tiff
code :
 SixLabors.ImageSharp.Image tmp= SixLabors.ImageSharp.Image.Load("aaa.png");
 tmp.SaveAsTiff("aa.tiff", new TiffEncoder()
 {
     BitsPerPixel = TiffBitsPerPixel.Bit32,
     Compression = TiffCompression.Lzw,
     CompressionLevel = SixLabors.ImageSharp.Compression.Zlib.DeflateCompressionLevel.BestCompression
 });
its work but lose alpha channel
png  pixel data is xxx xxx xxx 0
tiff pixel data is xxx xxx xxx 255
i have read source code ，seems encoder not support alpha ?

thanks in advance

**Comment by JimBobSquarePants** [comment, marked as answer]
@moonfanslyj Please do not create multiple discussion to ask the same question. 

https://github.com/SixLabors/ImageSharp/discussions/2830

Also please take the time to read up how to format markdown correctly. https://docs.github.com/github/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax

`ExtraSamples` as you have already identified is not currently supported by the encoder. https://github.com/SixLabors/ImageSharp/blob/21ec18e1b1186c3f1d549269271177d4516b3ba5/src/ImageSharp/Formats/Tiff/README.md

**Reply by moonfanslyj** [reply]
Sorry for creating two discussions .
I thought it was the problem, but after trying it it out, it wasn't，so I started a new discussion thread
thank you for your reply
